robot_sort/robot_sort.py

```diff
@@ -96,17 +98,21 @@ def sort(self):
         """
         Sort the robot's list.
         """
-        # Fill this out
-        pass
+    for i in range(len(l)-1):
+
+        for j in range(0, l-i-1):
+            if l[j] > l[j+1]:
+                swap_item(j)
```

Comment:
One section of the rubric gives points for implementation, and another section gives points just for planning, and typing out a plan in some pseudocode.

If you can't find a working solution, at least type out your thought process. Not only can it help type out the real code, but it can also give you some partial credit in interviews and on challenges like these.